types/i3dh-ios-bridge/i3dh-ios-bridge-tests.ts

```diff
@@ -0,0 +1,11 @@
+import { i3dh } from 'i3dh-ios-bridge';
```

Comment:
What's this package coming from?
Sorry, I cannot find a reference on NPM (I noticed non-npm)
How is this supposed to be used?

Reply:
These types are used internally by my company but also by their clients. It is supposed to define the API between the iOS ARKit and our product, so there are multiple possible packages where they could be used. Please do correct me if DefinitelyTyped is not the correct place for this kind of use case.

Reply:
are you shipping `package.json` with your content? I assume there is a non-publicly available package registry (say Azure, etc) used with custom npm config.
You can ship your types within your package, those types will be subject of types acquisition feature out of the box, as per documentation:
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package
let me know if this sounds like plausible scenario.
I do not see a added value for your company to ship and maintain declaration files separately

Reply:
After further consideration, I don't think it makes sense anymore to publish those types in DefinitelyTyped. Thank you for your input, I'll close this PR.